### PR TITLE
docs: Add VS Code WSL extension install step for dev env setup

### PR DESCRIPTION
**Pull request by smit9924**
- This commit adds a step to the CVAT development setup guide for users working with WSL (Windows Subsystem for Linux). The added instruction guides users to install the VS Code extension for WSL, ensuring that Visual Studio Code opens correctly inside the WSL environment.

- This change addresses an issue where users might encounter a 'DEBUG STOPPED' error if the extension is not installed, improving the overall setup experience.

- Related to issue #8313.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have created a changelog fragment 
- [x] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [x] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

## Summary by CodeRabbit

- **Documentation**
	- Added instructions for installing a Visual Studio Code extension for WSL to improve the development environment setup.

### CODE REVIEW

**Comment by smit9924** [comment]
@mdacoca can you please guide me know why linter fails. As much I understand is commit message has length more than 72 characters which I fixed. Now its showing me something like this "237:211  warning  Line must be at most 120 characters  maximum-line-length  remark-lint. Can you please kindly guide me.

**Comment by smit9924** [comment]
@azhavoro  can you please guide me know why Linter check fails. As much I understand is commit message has length more than 72 characters which I fixed. Now its showing me something like this "237:211 warning Line must be at most 120 characters maximum-line-length remark-lint. Can you please kindly guide me.

File: site/content/en/docs/contributing/development-environment.md [hunk 1]
```diff
@@ -234,7 +234,7 @@ You develop CVAT under WSL (Windows subsystem for Linux) following next steps.
   ```powershell
   wsl -d Ubuntu-18.04
   ```
-
+- Install the VS Code extension for WSL, which helps you to open VS Code correctly inside WSL. You can find the extension [here](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl).
```
Comment (azhavoro) [inline site/content/en/docs/contributing/development-environment.md:237]:
```suggestion
- Install the VS Code extension for WSL, which helps you to open VS Code correctly inside WSL.
  You can find the extension [here](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl).
```

**Review by azhavoro — APPROVED** [review]

**Comment by azhavoro** [comment]
Thanks for the contribution!

**Pull request merged**